packages/tools/replay-tool/src/helpers.ts

```diff
@@ -63,15 +63,13 @@ export function compareWithReferenceSnapshot(
      * Before replace - "{\"type\":\"https://graph.microsoft.com/types/map\",\"packageVersion\":\"0.28.0-214\"}"
      * After replace  - "{\"type\":\"https://graph.microsoft.com/types/map\",\"packageVersion\":\"X\"}"
      */
-    const packageVersionRegex = /\\"packageversion\\":\\".+\\"/gi;
+    const packageVersionRegex = /\\"packageversion\\":\\"[^"]+\\"/gi;
```

Comment:
What does this change?

Reply:
It's a little more accurate of a regex, since it's greedy it would capture additional properties in that line.

Reply:
I told it to exclude quotes " between the quotes. So now it would break if packageVersion had a " character in it, but this prevents the regex from grabbing additional properties.

Reply:
Makes sense.